Examples/Sources/MacroExamples/Implementation/ComplexMacros/DictionaryIndirectionMacro.swift

```diff
@@ -41,14 +40,14 @@ extension DictionaryStorageMacro: MemberAttributeMacro {
       return []
     }
 
-    return [
-      AttributeSyntax(
-        attributeName: IdentifierTypeSyntax(
-          name: .identifier("DictionaryStorageProperty")
-        )
+    var attribute = AttributeSyntax(
+      attributeName: IdentifierTypeSyntax(
+        name: .identifier("DictionaryStorageProperty")
       )
-      .with(\.leadingTrivia, [.newlines(1), .spaces(2)])
-    ]
+    )
+    attribute.leadingTrivia = [.newlines(1), .spaces(2)]
```

Comment:
I think this can be simplified further to 
```swift
    return [AttributeSyntax(
      leadingTrivia: [.newlines(1), .spaces(2)],
      attributeName: IdentifierTypeSyntax(
        name: .identifier("DictionaryStorageProperty")
      )
    )]
```